### PR TITLE
sci-visualization/paraview: version bump to 5.4.1

### PR DESCRIPTION
**Pull request by jok-ts**
Also fix build with dev-libs/jsoncpp-1.8.4

Bug: https://bugs.gentoo.org/644716
Bug: https://bugs.gentoo.org/643748

### CODE REVIEW

File: sci-visualization/paraview/paraview-5.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,285 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
[the remark this answers is not in the document]
Comment (jok-ts) [reply sci-visualization/paraview/paraview-5.4.1.ebuild]: Not being a gentoo dev I'm not sure what all is involved. All the phase functions from src_prepare on call the implementations from cmake-utils so I would've assumed the eclass takes care of the heavy lifting but it seems I'll need to rewrite every cmake-utils_use at least. Will try.

[the remark this answers is not in the document]
Comment (jok-ts) [reply sci-visualization/paraview/paraview-5.4.1.ebuild]: I think I got it, the configure phase still gives the same results as earlier apart from the changed default value of CMAKE_WARN_UNUSED_CLI. I haven't changed the ordering of the cmakeargs.

I'll push the changes as soon as I've verified it still builds.

**Comment by jok-ts** [comment]
@monsieurp Was there a problem with my changes three weeks ago?

**Comment by jok-ts** [comment]
@tamiko @monsieurp Is there any chance of getting this PR merged? It's been nine weeks already.

**Comment by jok-ts** [comment]
Ignored, closing.

**Comment by tamiko** [comment]
@jok-ts I am very sorry that I missed this one. :-/

**Pull request closed without merging**